enhancements/single-node/single-node-openshift-with-workers.md

```diff
@@ -0,0 +1,482 @@
+---
+title: single-node-openshift-with-workers
+authors:
+  - "@omertuc"
+reviewers:
+  - "@romfreiman"
+  - "@eranco74"
+  - "@tsorya"
+  - TBD
```

Comment:
you can please make it clear from this section what teams are going to be impacted, either by having to implement something for this now, or support the implementation in the future?  the teams should be tagged for review with an indication of what aspect of it they will be involved in.